src/lib/components/ManagePanel.js

```diff
@@ -55,9 +55,22 @@ export default class ManagePanel extends React.Component {
     return url;
   }
 
+  getDate(time) {
+    if (time === 'next') {
+      return 'Next time';
+    }
+    return moment(time).format('ddd, MMM D') || 'Later'
+  }
+
+  getTime(time) {
+    if (time === 'next') {
+      return 'Firefox opens';
+    }
+    return moment(time).format('[@] ha') || '';
+  }
```

Comment:
Uh, file a followup bug for that (in the Initial Release milestone), I think.  We'll need a new design from Sevaan.